LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -3211,7 +3211,7 @@ LongDescription="We have achieved a rudimentary understanding of ADVENT's roboti
 DisplayName="Muton Elite Autopsy"
 CodeName="Bluton"
 Summary="This behemoth is different enough from other Mutons that it merits a specific investigation."
-LongDescription="Central Officer Bradford has indicated that X-COM had intelligence reports of an elite form of muton that served as a guardian to the Elders. This creature appears to be that form. The genetic enhancements extend into the creature's brain, suggesting it is more able to recognize complex patterns and process information quickly in combat situations. R. Tygan."
+LongDescription="Central Officer Bradford previously mentioned old XCOM intelligence reports describing an elite form of Muton that served as a guardian to the Elders. This appears to be that form. The genetic enhancements extend into the creature's brain, suggesting improved pattern recognition and information processing in combat situations superior to the typical Muton. Additionally, the cadaver of this subject also appears to have several cybernetic implants forming a secondary endoskeleton which appear to boost the already extensive genetic modifications , some of which may be useful for our weapon development. I intend to let Shen investigate how to effectively use such contraptions, for my knowledge of prosthetics is far inferior to hers. -R. Tygan.\n\nOne of the more exciting cybernetics found in this type of Muton appears grafted to its brain. Initial testing shows that it can track targets within full spherical vision. While a human brain cannot process full 360 degree information in real time, we can create a tracking system that can guide itself to a targeted area. Luckily, during XCOM's initial operations to prep the Avenger for real combat, I've found several blueprints about a perfect Elerium-based warhead charge which our Technicals might make good use of. It'll take some time to miniaturize the implant, but we can make it work as the brain-computer interface components can be removed.\n\nEven more surprisingly, we just took a scan at the Muton Elite's body armor, and I got some ideas of improving our ballistic plating with the composite materials used in it. While we can't replicate its shield, we can come close enough with a \"carapace\" plating system. We can also use Dad's archives to make this work, as the original Alloy Carrier and TITAN-class armor prototypes would likely be able to be used to adapt this technology as a modular design. Most likely we will need to tear the components out of Muton Elite corpses, but with enough supplies I can make our operatives survive a plasma bolt in the field. -L. Shen."
```

Comment:
```suggestion
LongDescription="Central Officer Bradford previously mentioned old XCOM intelligence reports describing an elite form of Muton that served as a guardian to the Elders. This appears to be that form. The genetic enhancements extend into the creature's brain, suggesting improved pattern recognition and information processing in combat situations that are superior to the typical Muton. Additionally, the cadaver of this subject appears to have several cybernetic implants that boost the already extensive genetic modifications, some of which may be useful for our weapon development. I intend to let Shen investigate how to effectively make use of such systems, for my knowledge of cybernetics is far inferior to hers. R. Tygan.\n\nOne of the more exciting cybernetics found in this type of Muton appears grafted to its brain. Initial testing shows that it can track targets within full spherical vision. While a human brain cannot process full 360 degree information in real time, we can create a tracking system that can guide itself to a targeted area. Luckily, during XCOM's initial operations to prep the Avenger for real combat, I found several blueprints for a perfect Elerium-based warhead charge that our Technicals might make good use of. It'll take some time to miniaturize the implant, but we can make it work as the brain-computer interface components aren't needed for this use case.\n\nEven more surprisingly, we just took a scan of the Muton Elite's body armor and that gave me some ideas for improving our ballistic plating with the composite materials used in it. While we can't replicate the Muton's energy shield, we can come close enough with a \"carapace\" plating system. We can also use Dad's archives to make this work, as the original Alloy Carrier and TITAN-class armor prototypes can probably be used to adapt this technology to a modular design. Most likely we will need to tear the components out of Muton Elite corpses, but with enough supplies I can make our operatives survive a plasma bolt in the field. L. Shen."
```

Reply:
I tightened up the language in a few places. Let me know if it deviates too far from your vision. Also removed the dashes from the sign offs for consistency with the base game.